openedx/core/djangoapps/content/course_overviews/tests.py

```diff
@@ -164,6 +166,11 @@ def get_seconds_since_epoch(date_time):
             self.assertEqual(course_value, cache_miss_value)
             self.assertEqual(cache_miss_value, cache_hit_value)
 
+        # test tabs against course
+        course_overview_tabs = course_overview_cache_hit.course_overview_tab.all()
+        course_resp_tabs = [tab.tab_id for tab in course_overview_tabs]
+        self.assertEqual(self.COURSE_OVERVIEW_TABS, course_resp_tabs)
```

Comment:
@nasthagiri `course_overview_tabs` contains `CourseOverviewTab` Model objects set fetched via DB query as `course_overview` does not include any tabs field; we created a new related model to save this info. It is not just a set of direct accessible tabs data that could be compared either with list or set. 
If we do create expected Model objects to compare results it won't have primary keys and the tests won't pass.
Let me know if I am not going in the right direction.